backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/process/adprocess/ADProcessPostProcessService.java

```diff
@@ -415,7 +408,7 @@ else if (recordsToOpen != null && recordsToOpen.getTarget() == OpenTarget.Single
 			final DocumentPath documentPath = extractSingleDocumentPath(recordsToOpen);
 			return OpenSingleDocument.builder()
 					.documentPath(documentPath)
-					.modal(true)
```

Comment:
pls precisely set the targetTab, even if we have a default defined.
This will make the code more readable and robust.